content/blog/2019-09-25-bril-to-llvm-ocaml.md

```diff
@@ -124,7 +124,7 @@ v: bool = const true;
 print v;
 ```
 
-This would actually be one potential reason to implement an SSA transformation of Bril before transforming to LLVM (some LLVM optimizations can usually optimize away the stack so for the sake of code efficiency it wasn't strictly necessary). If Bril was in SSA form, then every variable would only have 1 type and we could unambiguously figure out which print to use.
+There is some discussion of modifying the Bril spec to make the above code snippet illegal. Therefore, in this project I was comfortable assuming a static type for each variable.
```

Comment:
I realized the previous statement was incorrect, so I changed it to what we were talking about.